lua/lib/lib.lua

```diff
@@ -366,4 +367,16 @@ function M.toggle_dotfiles()
   return M.refresh_tree()
 end
 
+function M.dir_up(node)
+  if not node then
+    return M.change_dir('..')
+  else
+    local newdir = vim.fn.fnamemodify(node.absolute_path, ':h')
+    if newdir == M.Tree.cwd then
+      M.change_dir('..')
+    end
+    return M.set_index_and_redraw(newdir)
```

Comment:
I find it confusing when going up a dir it closes the expanded path to the current node and jumps the cursor to the top of the tree, i'd rather see it this way
```suggestion
    return M.set_index_and_redraw(node.absolute_path)
```
what do you think?